packages/plugin-phone/test/integration/spec/call.js

```diff
@@ -223,47 +320,161 @@ describe(`plugin-phone`, function() {
       });
 
       describe(`when the call ends`, () => {
-        it(`gets revoked`);
+        it(`gets revoked`, () => {
+          let localMediaStreamUrl;
+          let revokeSpy;
+          const call = spock.spark.phone.dial(mccoy.email);
+          assert.equal(call.remoteMediaStream, null);
+          return Promise.all([
+            mccoy.spark.phone.when(`call:incoming`)
+              .then(([c]) => {
+                return c.answer();
+              }),
+            call.when(`connected`)
+              .then(() => {
+                localMediaStreamUrl = call.localMediaStreamUrl;
+                return assert.isDefined(localMediaStreamUrl);
+              })
+              .then(() => {
+                revokeSpy = sinon.spy(URL, `revokeObjectURL`);
+                return call.hangup();
+              }),
+            call.when(`disconnected`)
+              .then(() => assert.called(revokeSpy))
+          ]);
+        });
       });
     });
 
     describe(`#sendingAudio`, () => {
+      let call;
+      beforeEach(() => {
+        call = spock.spark.phone.dial(mccoy.email);
+        return Promise.all([
+          mccoy.spark.phone.when(`call:incoming`)
+            .then(([c]) => c.answer()),
+          call.when(`connected`)
+            .then(() => assert.equal(call.status, `connected`))
+        ]);
+      });
+
+      afterEach(() => {
+        const c = call;
+        call = undefined;
+        return c.hangup().catch((reason) => console.warn(reason));
+      });
+
       describe(`when the local party is sending Audio`, () => {
-        it(`is true`);
+        it(`is true`, () => {
+          return call.startSendingAudio()
+            .then(() => assert.isTrue(call.sendingAudio));
+        });
       });
 
       describe(`when the local party is not sending Audio`, () => {
-        it(`is false`);
+        it(`is false`, () => {
+          return call.stopSendingAudio()
+            .then(() => assert.isFalse(call.sendingAudio));
+        });
       });
     });
 
     describe(`#sendingVideo`, () => {
+      let call;
```

Comment:
Does it make sense to do these tests based on starting the calls with different constraints rather instead of changing the audio status in the call?